packages/trader/src/Modules/Reports/Components/unavailable-contract.jsx

```diff
@@ -0,0 +1,42 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import { Localize } from '@deriv/translations';
+import { urlFor } from '@deriv/shared';
+import { website_name } from 'App/Constants/app-config';
+
+const contract_type_binary_url = [
+    'user/portfoliows', // Open position - 0
+    'user/profit_tablews', // Profit table  - 1
+    'user/statementws', // Statement     - 2
+];
+
+const UnavailableContract = ({ contract_type, is_eu }) =>
```

Comment:
Think it would be more clear if we pass contract_type as a string instead?

Reply:
As we've put some comments on the target component, I think it should suffice.

Reply:
Currently you have to look inside the component to find out what `0` `1` `2` means - it would be easier to understand the components usage if it was a string instead of `0` - which would be directly clear if the prop is e.g. `contract_type_binary_url={'user/portfoliows'}` instead.

If you want to to make it only accept these 3 strings (and nothing else) it's possible to use a proptype check to turn it into an enum - with Proptypes.string.oneOf(['string', 'string'])

Reply:
Yea. Maybe better to change the prop and directly sending the URL.